pkg/validationfile/blocks/relationships.go

```diff
@@ -54,11 +54,11 @@ func (pr *ParsedRelationships) UnmarshalYAML(node *yamlv3.Node) error {
 			)
 		}
 
-		_, ok := seenTuples[tuple.String(tpl)]
+		_, ok := seenTuples[tuple.StringWithoutCaveat(tpl)]
```

Comment:
👍🏻 we want to ignore dupes. I wonder if instead of ignoring we want to report an error?